mlflow/tracking/client.py

```diff
@@ -972,11 +992,12 @@ def log_text(self, run_id, text, artifact_file) -> None:
                 f.write(text)
 
     @experimental
-    def log_dict(self, run_id, dictionary, artifact_file) -> None:
+    def log_dict(self, run_id: str, dictionary: Any, artifact_file: str) -> None:
```

Comment:
Used `Any` for `dictionary` becasue the `json`module does the same thing:

![Screen Shot 2021-04-19 at 17 19 43](https://user-images.githubusercontent.com/17039389/115204575-98915f00-a133-11eb-83ef-fa83718621bf.png)